src/main/java/io/javalin/http/util/ContextUtil.kt

```diff
@@ -40,7 +40,7 @@ object ContextUtil {
     fun urlDecode(s: String): String = URLDecoder.decode(s.replace("+", "%2B"), "UTF-8").replace("%2B", "+")
 
     fun getBasicAuthCredentials(header: String?): BasicAuthCredentials? = try {
-        val (username, password) = String(Base64.getDecoder().decode(header!!.removePrefix("Basic "))).split(":")
+        val (username, password) = String(Base64.getDecoder().decode(header!!.removePrefix("Basic "))).split(":", 2)
```

Comment:
It's actually missing arg 2, 
```
    vararg delimiters: String, 
    ignoreCase: Boolean = false, 
    limit: Int = 0
```
Needs named params or add ignoreCase param

Reply:
Even though https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/split.html shows it be possible with only two. @tipsy probably knows what's up.

Reply:
Yeah, I noticed this. I'm not a kotlin guy so I'll wait for @tipsy.
If he doesn't get to it later today, I'll see if I can actually run it locally.

Reply:
Use `.split(':', limit = 2)`, the ignoreCase is irrelevant as we aren't splitting on alphabetical character.